osutil/disks/disks_linux.go

```diff
@@ -106,35 +117,43 @@ func parseUdevProperties(r io.Reader) (map[string]string, error) {
 	return m, scanner.Err()
 }
 
-// DiskFromDeviceName finds a matching Disk using the specified name, such as
```

Comment:
Should the new `DiskFromDevicePath` also have a doc-string similar to the DiskFromDeviceName ?